concepts/subscribers.mdx

```diff
@@ -191,3 +191,152 @@ Novu will create a subscriber if one does not exist and will update the existing
 
 You can create subscribers in bulk _(up to 500 at once)_ via the SDKs or [API](/api-reference/subscribers/bulk-create-subscribers).
 
+## Export Subscribers
+
+Currently, Novu does not support exporting subscribers. However, [Get Subscribers API](/api-reference/subscribers/get-subscribers) can be used to fetch subscribers
+
+<Tabs>
+<Tab title="Node.js">
+### Steps:
+- `mkdir export_novu_subscribers`
+- `cd export_novu_subscribers`
+- Save below code in `fetch_subscribers.js` file
+- `npm init -y`
+- `npm install axios fast-csv fs`
+- Run `node fetch_subscribers.js`
+- Subscribers are exported in `subscribers.csv` file
+
+```javascript
+const axios = require("axios");
+const fs = require("fs");
+const csv = require("fast-csv");
+
+// Use https://eu.api.novu.co/v1/subscribers for eu region
+const API_URL = "https://api.novu.co/v1/subscribers";
+
+// Replace with actual API key
+const API_KEY = "API_KEY";
+
```

Comment:
Can we move this to a github `gist` and just add a link here? I don't want to go so level at this page for an edge case

Reply:
@scopsy added gist links